src/chrome/content/rules/GlobalConnect.dk.xml

```diff
@@ -0,0 +1,23 @@
+<!--
+  Problematic domains:
+
+    - ^ ⁴
+
+  ¹: Works but serves a lot of strict http content
+  ²: 403
+  ³: connection timeout
+  ⁴: refused
+-->
+<ruleset name="GlobalConnect.dk (partial)">
+
+	<target host="globalconnect.dk" />
+	<target host="www.globalconnect.dk" />
+
+    <rule from="^http:"
```

Comment:
```
    <!-- Rule to avoid connection refused on non-www website root -->
    <rule from="^http://globalconnect\.dk"
        to="https://www.globalconnect.dk" />

    <rule from="^http:" to="https:" />
```